### PR TITLE
switch travel time to arrival time of a player, update test

### PR DESCRIPTION
**Pull request by YIllmann**
see title.

@n00on @lukas0820 I changed the attribute in the api.yml as well as the describtion.

### CODE REVIEW

**Comment by n00on** [comment]
@YIllmann did you maybe forget to checko in the api.yml? I don't see it in the diff

**Comment by YIllmann** [comment]
> @YIllmann did you maybe forget to checko in the api.yml? I don't see it in the diff

that may be correct :D. Update ist pushed

**Comment by lukas0820** [comment]
@YIllmann there is a naming conflict between api.yaml and lobby.py. In api.yaml, the parameter is named scenario_start_time, while in lobby.py, it is referred to as starting_time.

I assume that starting_time might be the correct one

**Comment by lukas0820** [comment]
I`ve noticed that the starting time is in milliseconds and the arrival time in seconds in api.yaml. Is this correct or should it both be seconds?

File: server/execution/entities/player.py [hunk 1]
```diff
@@ -20,8 +21,19 @@ def __init__(self, tan: str, name: str | None, alerted: bool,
         self.accessible_locations = accessible_locations
         self.logged_in = logged_in  # is toggled upon first login
 
+        self.alerted_timestamp = 0
```
Comment (simohlsen) [inline server/execution/entities/player.py:24]: Would set it to `-1`

File: server/execution/entities/player.py [hunk 2]
```diff
@@ -20,8 +21,19 @@ def __init__(self, tan: str, name: str | None, alerted: bool,
...
 
+    def alert(self):
+        if not self.alerted:
+            self.alerted = True
+            self.alerted_timestamp = time.current_time_s()
+
+    def remove_alert(self):
+        if self.alerted:
+            self.alerted = False
+            self.alerted_timestamp = 0
```
Comment (simohlsen) [inline server/execution/entities/player.py:35]: Would set it to `-1`

**Review by simohlsen — CHANGES_REQUESTED** [review]

**Comment by YIllmann** [comment]
> I`ve noticed that the starting time is in milliseconds and the arrival time in seconds in api.yaml. Is this correct or should it both be seconds?

both should seconds.

**Review by simohlsen — APPROVED** [review]

**Pull request merged**